elasticdl/python/common/model_handler.py

```diff
@@ -202,6 +202,8 @@ def get_model_to_train(self, model):
         """Replace the tf.keras.layers.Embedding layer in the model with
         an elasticdl.layers.Embedding layer in ParameterServerStrategy.
         """
+        # clear keras model session to avoid clutter from old models/layers.
+        tf.keras.backend.clear_session()
```

Comment:
The old model is only defined before but not used. So the model definition will also have some side effects?

Reply:
The side effects may be that the model weights are created repeatedly and the memory is wasted.